src/main/flight/pid_init.c

```diff
@@ -423,6 +423,8 @@ void pidInitConfig(const pidProfile_t *pidProfile)
 #endif
 
     pidRuntime.levelRaceMode = pidProfile->level_race_mode;
+    pidRuntime.tpaBreakpoint = (pidProfile->tpa_breakpoint - 1000) * 0.001f; // .35 from 1350, range can be 0 to 0.99, limited by CLI range
```

Comment:
```suggestion
    pidRuntime.tpaBreakpoint = (pidProfile->tpa_breakpoint - 1000) * 0.001f;      // .35 from 1350, 
    pidRuntime.tpaBreakpoint = constrainf(pidRuntime.tpaBreakpoint, 0.0f, 0.99f); // range can be 0 to 0.99
```

Reply:
OK, so if someone sets the TPA_BREAKPOINT in CLI to a value above 1990, they get 1990, effectively.
This solves the divide by zero possibility; I'll update as suggested. 
Thanks.